src/components/interaction-designer/block-editors/GroupMembershipEditor.vue

```diff
@@ -28,14 +29,15 @@
         <template #input-control="{isValid}">
           <vue-multiselect
             v-if="isGroupListVisible"
-            v-model="selectedGroups"
             :class="{invalid: isValid === false}"
             :is-loading="hasGroupsLoading"
             :options="groupOptions"
             :taggable="!availableGroups"
             :multiple="true"
             track-by="group_key"
             label="group_name"
+            :value="selectedGroups"
+            @input="setSelectedGroups"
             @seach-change="onSearchChange"
```

Comment:
@object-required there is a typo in `@search-change` which probably means this is not used anywhere. Should we delete the related downstream logic? Or keep it in case the lib's users need this functionality?